src/swapDataBlock.js

```diff
@@ -0,0 +1,97 @@
+import { EditorState } from "draft-js";
```

Comment:
Hi @mogavin, Could you please put the copyright holder and license inside this file?

Reply:
Done !